WooCommerce/Classes/ViewRelated/MainTabBarController.swift

```diff
@@ -725,18 +726,19 @@ private extension MainTabBarController {
     }
 
     func observeSiteIDForViewControllers() {
-        cancellableSiteID = stores.siteID.sink { [weak self] siteID in
+        cancellableSiteID = stores.site.sink { [weak self] site in
```

Comment:
Super nit: should the method and the property be renamed to `~site` instead of `~siteID`?